### PR TITLE
Attempt to clarify open source vs. freemium vs. paid

### PR DESCRIPTION
**Pull request by joeduffy**
This change:

- Add the Community Edition to the top-level edition tabs on
  the pricing page, replacing the separate section we already had.

- Adjust the feature differentiation waterfall in the edition tabs.

- Add a section to the top of the pricing page that clarifies that
  Pulumi is available as open source, freemium, or paid editions.

- Update the FAQ to call out the ability to use the open source
  independent of the service.

- Updates some text so that it doesn't sound like we're limiting
  the Community Edition to "individuals" -- in reality, it really
  just means "single user scenarios", which is subtly different.

This is certainly not perfect, but is a set of incremental changes
that I hope will clarify things at least a little bit.

The only thing I don't love is that it pushes the pricing tabs
closer to the fold, but I've adjusted some margins and kept the
new text fairly short, in an attempt to mitigate this.

### CODE REVIEW

File: layouts/page/pricing.html [hunk 1]
```diff
@@ -1,55 +1,114 @@
...
+                </div>
+            </div>
         </div>
     </header>
 {{ end }}
 
 {{ define "main" }}
-    <section class="bg-purple-800 pt-24 text-white px-4 lg:px-0">
+    <section class="bg-purple-800 pt-24 text-white px-4 lg:px-0 justify-center text-center">
+        <h2 class="text-gray-300 mb-8">Pulumi Editions</h2>
```
Comment (cnunciato) [inline layouts/page/pricing.html]: You could win back a little vertical space by dropping this, since it's not really necessary.

Comment (joeduffy) [reply layouts/page/pricing.html]: Sounds great. Done!

**Comment by mikhailshilkov** [comment]
This is a great change! Here are some ideas to bring it even further.

My initial screen at the laptop looks like this:
image

There's no pricing table yet, I only see two blocks of text. There seems to be enough space to add more info here.

The transition to the pricing tiers isn't fully. If it's open-source, why is it not free for teams?

I think "you can run Pulumi offline and manage state yourself" might not be clear enough.

I would consider making three statements:

- Pulumi CLI is free
- Pulumi backend is not free but useful for teams
- Pulumi backend is free for individuals

The first FAQ answer is great!

That's my impression, but overall I'm super happy with the direction of the change.

**Comment by joeduffy** [comment]
Great feedback, @mikhailshilkov!

@cnunciato I agree with a lot of the feedback, but honestly struggled to get the layout close to what I had originally envisioned, especially as I wanted to be consistent with our overall hero styling (which, frankly, probably needs to be revisited overall given how much wasted sea of purple space it has). Completely random example of another website, https://unearthlabs.com/pricing/, but notice how much information is instantly visible above the fold.

I'll keep pondering -- I am thinking of maybe merging it close to as-is, since it's a good step forward, and then perhaps we can file an issue for a deeper overhaul during M31?

**Comment by cnunciato** [comment]
I’m +1 to merge even as-is because it addresses the important bit by clarifying the existence of a first-class free plan. I’m happy to work on incorporating this feedback in a follow-up pass.

**Comment by joeduffy** [comment]
Here's an alternative I was just toying with. Better? Worse?

image

**Comment by joeduffy** [comment]
One thing I dislike is the "run it offline" link may distract from the CTAs on the page.

**Comment by joeduffy** [comment]
@cnunciato I propose we keep this change simple for now, esp. given the potentially significant impact to user behavior on this page. This effectively now comes down to not burying Community Edition at the bottom of the page. Notably, this doesn't really clarify the relationship between Pulumi's SDK open source and the fact you can use the service (or not), which is still a problem.

image

As a bonus, it actually saves a dozen or so vertical pixels compared to our current page :smile:

**Comment by cnunciato** [comment]
Love it. I was actually going to suggest we minimize the changes to the hero section just to keep from switching up too many things at once. This is great. :shipit:

**Pull request merged**